src/aznfswatchdog

```diff
@@ -122,22 +149,57 @@ reconcile_conntrack()
             if [[ "$line" =~ $matchstr ]]; then
                 l_sport=${BASH_REMATCH[1]}
                 l_dport=${BASH_REMATCH[2]}
-                reconcile_conntrack_one $l_ip $l_sport $l_dport $l_nfsip
+                reconcile_conntrack_one $l_ip $l_sport $l_dport $l_nfsip "SYN_SENT"
+            fi
+        done
+    fi
+
+    IFS=$'\n' output111=$(conntrack -L -p tcp -d $l_ip --dport 111 --state ESTABLISHED 2>/dev/null)
+    IFS=$'\n' output2048=$(conntrack -L -p tcp -d $l_ip --dport 2048 --state ESTABLISHED 2>/dev/null)
+    output="$output111"$'\n'"$output2048"
+    if [ -n "$output" ]; then
+        for line in $output; do
+            # XXX Remove this log after running for few days.
+            vecho "$line"
+
+            matchstr=".* ESTABLSIHED src=([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+) dst=$l_ip sport=([0-9]+) dport=([0-9]+) [UNREPLIED].*"
+            if [[ "$line" =~ $matchstr ]]; then
+                l_sysip=${BASH_REMATCH[1]}
```

Comment:
this is the source IP so this will be the interface IP, hence this is not very interesting. Also u cannot use this at L90 since you use -r there which means source in the reply direction.
What is more interesting is the src after UNREPLIED. This is the source in the reply direction and will be useful for matching.
save that as l_reply_srcip and then you can use it at L90